lib/Parser/LineChecker.php

```diff
@@ -43,15 +37,18 @@ public function isSpecialLine(string $line): ?string
         return $letter;
     }
 
-    public function isListLine(string $line, bool $isCode): bool
```

Comment:
I'm not sure what the `$isCode`'s function was here. Code blocks shouldn't be subparsed, so this should never be true. So I simply removed it.